appmetrica/push/api.py

```diff
@@ -23,31 +23,40 @@ class TokenTypes(object):
 class PushAPI(BaseAPI):
     base_url = 'https://push.api.appmetrica.yandex.net/push/v1/'
 
-    def create_group(self, name):
+    def create_group(self, name, send_rate=None):
         """
         Create group to combine the sending in the report
         :param name: Unique name of the group
+        :param send_rate: Limit on the max speed of sending push messages (number per second)
         :return: Identifier of the created group
         """
         assert name
-        data = {
-            'group': {
-                'app_id': self.app_id,
-                'name': name
-            }
+        group = {
+            'app_id': self.app_id,
+            'name': name,
         }
+        if send_rate is not None:
+            if not isinstance(send_rate, int):
+                logger.error('create_group with send_rate = %s failed. must be an integer', send_rate)
+                raise exceptions.AppMetricaCreateGroupError
+            if send_rate < 100 or send_rate > 5000:
```

Comment:
```suggestion
            if not (100 <= send_rate <= 5000):
```
Так было бы нагляднее имхо